### PR TITLE
update details and add internal namespace

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

**Review by Jinming-Hu — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs [hunk 1]
```diff
@@ -199,7 +200,7 @@ private static void BuildNodes(CodeFileTokensBuilder builder, List<NavigationIte
             {
                 // Filter namespace based on file name if any of the namespace matches file name pattern
                 // If no namespace matches file name then allow all namespaces to be part of review to avoid mandating file name convention
-                if ((!foundFilterNamespace || nameSpace.StartsWith(packageNamespace)) && !nameSpace.EndsWith(DetailsNamespacePostfix))
+                if ((!foundFilterNamespace || nameSpace.StartsWith(packageNamespace)) && !(nameSpace.EndsWith(DetailsNamespacePostfix) || nameSpace.EndsWith(InternalNamespacePostfix)))
```
Comment (RickWinter) [inline src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs:203]: The internal namespace should stay in the APIView.  Although we do not expect customers to use it, we do have a bar that it cannot have breaking changes.

**Review by RickWinter — DISMISSED** [review]

File: src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs (hunk 1 above)
Comment (ahsonkhan) [reply src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs]: Yes, please! Thanks for picking that up @RickWinter

**Comment by ahsonkhan** [comment]
> Removed the internal namespace change. Left only the detail namespace update

Can you verify that public types continue to show up even if they do derive from something in `_detail`. It might be a good idea if that inheritance was also visible, but might be trickier to achieve.

**Review by RickWinter — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs [hunk 2]
```diff
@@ -39,7 +39,7 @@ public class CppLanguageService : LanguageService
         private const string AccessModifierProtected = "protected";
         private const string AccessModifierPublic = "public";
         private const string RootNamespace = "Azure";
-        private const string DetailsNamespacePostfix = "::Details";
```
Comment (sima-zhu) [inline src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs:42]: Is "Details" here still in use anywhere? Just want to make sure it is not breaking anything

Comment (RickWinter) [reply src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs:42]: No, this is an internal namespace only.  It won't break anything.

**Pull request merged**